libs/ui-web/README.md

```diff
@@ -4,6 +4,11 @@
 
 ![@orchestrator/ui-web](https://img.shields.io/npm/v/@orchestrator/ui-web)
 
+**DEPRECATED!**
+
+> Please use [`@orchestrator/html-tag`](/libs/html-tag) package instead
+> which has more streamlined API and allows to render any HTML tags available in the Web.
```

Comment:
I wonder should we deprecate it or not, what if oneday we will want to provide complex components, like counters, spinners and others.

Reply:
The ui-web lib was only meant to have native html elements as components which this lib is replacing.
So if we would want to create a new component we will create a new package for it (like stepper).